ArduPlane/quadplane.cpp

```diff
@@ -1281,18 +1302,19 @@ void QuadPlane::update_transition(void)
      */
     if (have_airspeed &&
         assistance_needed(aspeed) &&
-        !is_tailsitter() &&
         hal.util->get_soft_armed() &&
         ((plane.auto_throttle_mode && !plane.throttle_suppressed) ||
          plane.get_throttle_input()>0 ||
          plane.is_flying())) {
         // the quad should provide some assistance to the plane
-        if (transition_state != TRANSITION_AIRSPEED_WAIT) {
```

Comment:
this change seems to have no effect, why change it?

Reply:
to enable assisted flight for tail sitters without entering airspeed wait transition state